core/arch/arm/kernel/kern.ld.S

```diff
@@ -172,6 +172,12 @@ SECTIONS
 		__extab_end = .;
 	}
 
+	/*
+	 * .got is normally read/write but once relocation is done it's
+	 * more safe to make it read-only.
+	 */
+	__rodata_end = .;
```

Comment:
So now `__rodata_start` / `__rodata_end` do not represent the start and end of the `.rodata` section anymore, but the start and end of the read-only sections (well, those we want to be read-only after initializations). It's a bit misleading. How about `__ro_sections_start` / `__ro_sections_end` instead?

Reply:
I'll fix.